src/plattenalbum.py

```diff
@@ -1038,8 +1038,8 @@ class SetupDialog(ConnectDialog):
 	def __init__(self):
 		super().__init__(_("Setup"), GLib.Variant("b", False))
 		box=Gtk.Box(orientation=Gtk.Orientation.VERTICAL, spacing=12)
-		box.append(Gtk.Label(label=_("After installing the “Music Player Daemon” (<tt>mpd</tt>) the following two commands"\
```

Comment:
Ok, so the quotes are unnecessary? When are they actually needed? I think at least for abbreviations like "MPRIS" they should remain. (I haven't added them around mpd here since this should just be the package name in most cases.)

Reply:
It’s a proper name, so I wouldn’t use quotation marks there myself. I don’t think it’s too big of a deal, though, both with and without works.